mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Kitto.Mixfile do
     [{:cowboy, "~> 1.0.0"},
      {:plug, "~> 1.3"},
      {:poison, "~> 3.0"},
-     {:fs, "~> 2.11.0"},
+     {:fs, github: "synrc/fs"},
```

Comment:
@davejlong did you delete a comment here?
I have this on my feed, strange..
![selection_076](https://cloud.githubusercontent.com/assets/645514/21272658/ea348f00-c3c8-11e6-9769-b6ecca15b3c3.png)